### PR TITLE
Added cutandrun specific memory reqs for unc_longleaf.config

### PR DESCRIPTION
**Pull request by ahepperla**
---
name: New Config
about: A new cluster config
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [ ] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [ ] Add your custom config file to the `conf/` directory
- [ ] Add your documentation file to the `docs/` directory
- [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [ ] Add your custom profile to the `README.md` file in the top-level directory
- [ ] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

**Review by jfy133 — COMMENTED** [review]
Sorry, was on my phone earlier and wasn't completely clear and helpful:

Please see the docs here - https://github.com/nf-core/configs?tab=readme-ov-file#adding-a-new-pipeline-specific-config

It's a similar process to the insittutional configs

Sorry to make it a bit more convoluted 😅

**Comment by ahepperla** [comment]
> Sorry, was on my phone earlier and wasn't completely clear and helpful:
> 
> Please see the docs here - https://github.com/nf-core/configs?tab=readme-ov-file#adding-a-new-pipeline-specific-config
> 
> It's a similar process to the insittutional configs
> 
> Sorry to make it a bit more convoluted 😅

All good @jfy133 I just need to read more thoroughly LOL. I think I did it correctly this time...?

**Review by pontus — APPROVED** [review]

File: conf/pipeline/cutandrun/unc_longleaf.config [hunk 1]
```diff
@@ -0,0 +1,17 @@
+params {
+    config_profile_description = "BARC nf-core profile for UNC's Longleaf HPC."
```
Comment (jfy133) [inline conf/pipeline/cutandrun/unc_longleaf.config:2]:
```suggestion
    config_profile_description = "cutandrun BARC nf-core profile for UNC's Longleaf HPC."
```

**Review by jfy133 — APPROVED** [review]
Once this is in we will need to make a quick pull request to cutandrun to activate the institutional pipeline-specific config fiunctionality

Basically these lines need to be un commented: https://github.com/nf-core/cutandrun/blob/6e1125d4fee4ea7c8b70ed836bb0e92a89e3305f/nextflow.config#L168-L173

**Pull request merged**